src/main/java/edu/cornell/kfs/pdp/document/authorization/CuPayeeACHAccountMaintenanceDocumentAuthorizer.java

```diff
@@ -0,0 +1,27 @@
+package edu.cornell.kfs.pdp.document.authorization;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.kim.api.identity.Person;
+import org.kuali.kfs.krad.document.Document;
+import org.kuali.kfs.pdp.document.authorization.PayeeACHAccountMaintenanceDocumentAuthorizer;
+
+public class CuPayeeACHAccountMaintenanceDocumentAuthorizer extends PayeeACHAccountMaintenanceDocumentAuthorizer {
+    private static final Logger LOG = LogManager.getLogger();
+
+    @Override
+    public boolean canApprove(Document document, Person user) {
+
+        try {
+            String documentInitializerPrincipalId = document.getDocumentHeader().getWorkflowDocument().getInitiatorPrincipalId();
```

Comment:
For consistency with KualiCo's terminology, I would suggest renaming this variable to say "Initiator" instead of "Initializer".